README.md

```diff
@@ -63,9 +63,10 @@ There are different ways to get the Electron desktop app:
 
 * Get the official builds from [here, in GitHub Releases][gh-releases]
 for any platform.
-* Get it from AUR for Arch/Manjaro with `yay -S dogehouse`.
+* Get it from AUR (unofficial package) for Arch/Manjaro with
+`yay -S dogehouse`.
 * Get the desktop client for Debian-based distros (including Ubuntu)
-from the official APT repo with these simple steps:
+from the unofficial APT repo with these simple steps:
```

Comment:
ok no ben said its not official but if it contains his domain you can call it official, so change this back

Reply:
Also you duped your review comments lmao.